### ISSUE TITLE
Problem getting sentences/plaintext from a section

### ISSUE THREAD

**Issue by ivanmkc** [spencermountain/wtf_wikipedia#132]
Hi,

I'm using this for the first time and wrote a quick test script to get section text:

```
const wtf 				= require('wtf_wikipedia');

wtf.fetch('Albert_Einstein', 'en', function(err, doc) {
  var sentences = doc.section(0).sentences();
  console.log(sentences)
});
```

Seems like the first few sentences (and the last) of section 0 are parsed incorrectly as they contain Wikitext from the infobox. Also, it seems to affect all the other sections, for example, doing the same for section 1 returns zero sentences (when there should be as verified via the official API).

Am I using this incorrectly or is the Albert_Einstein article a little too complex for the parser? FYI, it's been a while since I used node.js as well.

Thanks!

**Comment by spencermountain** [comment]
you're right!
thanks Ivan, will take a look

**Comment by spencermountain** [comment]
fixed in v4.5.0
thanks

**Comment by ivanmkc** [comment]
Thanks, awesome response time and I can confirm it works
